Add troubleshooting for pg-gvm errors in logs (#117)
greenbone/docs: src/21.04/container/index.md

    restart gvmd
```

### Errors while starting `pg-gvm` container

> While starting up the `pg-gvm` container some errors are displayed. For example

Reviewer: ```suggestion
While starting up the `pg-gvm` container, some errors are displayed. For example
```

`createuser: error: creation of new role failed: ERROR:  role "gvmd" already exists`
or `ERROR:  extension "uuid-ossp" already exists`.

> The code behind these errors tries to setup the database. If the database is

Reviewer: ```suggestion
The code behind these errors tries to set up the database. If the database is
```

already initialized, all tables, users, permissions and extensions exist,

> errors are raised. At the moment it is not possible to silence these errors but

Reviewer: ```suggestion
errors are raised. At the moment, it is not possible to silence these errors but
```
